server/src/main/java/org/tango/server/servant/AttributeGetterSetter.java

```diff
@@ -231,6 +231,10 @@ static void setAttributeValue(final fr.esrf.Tango.AttributeValue[] values, final
                 if (e.getCause() instanceof DevFailed) {
                     back[i] = TangoIDLAttributeUtil.toAttributeValue5Error(names[i], AttrDataFormat.FMT_UNKNOWN, 0,
                             (DevFailed) e.getCause());
+                } else {
```

Comment:
Probably we do need this if here. If *CacheException* is a *DevFailed* we just need 2 **catch** clauses here:

```java
catch(CacheException e){...}
catch(DevFailed e){...}

```

Reply:
If you want.